context.js

```diff
@@ -140,9 +140,9 @@ class Context {
     return this
   }
 
-  unexpectedError (err) {
+  unexpectedError (err, formatted) {
```

Comment:
Let's remove the `formatted` argument here and format the given error in this method instead.